src/main/java/dev/emi/trinkets/api/TrinketComponent.java

```diff
@@ -1,27 +1,31 @@
 package dev.emi.trinkets.api;
 
-import java.util.List;
-import java.util.function.Predicate;
-
 import dev.emi.trinkets.api.Trinket.SlotReference;
 import dev.onyxstudios.cca.api.v3.component.ComponentV3;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.Pair;
 
+import java.util.List;
+import java.util.function.Predicate;
+
 public interface TrinketComponent extends ComponentV3 {
 
-	public TrinketInventory getInventory();
+	TrinketInventory getInventory();
 
-	public boolean isEquipped(Predicate<ItemStack> predicate);
+	boolean isEquipped(Predicate<ItemStack> predicate);
 
-	public default boolean isEquipped(Item item) {
+	default boolean isEquipped(Item item) {
 		return isEquipped(stack -> stack.getItem() == item);
 	}
 
-	public List<Pair<SlotReference, ItemStack>> getEquipped(Predicate<ItemStack> predicate);
+	List<Pair<SlotReference, ItemStack>> getEquipped(Predicate<ItemStack> predicate);
 
-	public default List<Pair<SlotReference, ItemStack>> getEquipped(Item item) {
+	default List<Pair<SlotReference, ItemStack>> getEquipped(Item item) {
 		return getEquipped(stack -> stack.getItem() == item);
 	}
+
+	default List<Pair<SlotReference, ItemStack>> getAllEquipped() {
+		return getEquipped(stack -> true);
```

Comment:
This is going to return all slots, even empty ones, is this intended behavior or would `stack -> !stack.isEmpty()` be a better option?

Reply:
Good catch